compiler+runtime/src/cpp/jank/runtime/obj/character.cpp

```diff
@@ -81,23 +47,21 @@ namespace jank::runtime
 
   void obj::character::to_string(fmt::memory_buffer &buff) const
   {
-    /* TODO: This is actually to_representation, since the string version of \a is just a. */
-    fmt::format_to(std::back_inserter(buff), "{}", get_char_from_literal(data).unwrap());
+    fmt::format_to(std::back_inserter(buff), "{}", data);
   }
 
   native_persistent_string obj::character::to_string() const
   {
-    auto const char_repr{get_char_from_literal(data).unwrap()};
-    return native_persistent_string{1, char_repr};
+    return data;
   }
 
   native_persistent_string obj::character::to_code_string() const
   {
-    return fmt::format("\\{}", data);
+    return get_literal_from_char(data[0]);
   }
 
   native_hash obj::character::to_hash() const
   {
-    return hash::visit(get_char_from_literal(data).unwrap());
+    return hash::visit(data[0]);
```

Comment:
We want to hash the whole string. This will be important for multi-byte chars (i.e. unicode support).